src/rez/resolved_context.py

```diff
@@ -233,7 +233,7 @@ def __init__(self, package_requests, verbosity=0, timestamp=None,
         self.package_filter = (PackageFilterList.singleton if package_filter is None
                                else package_filter)
 
-        self.package_orderers = (
+        self.package_orderers = PackageOrderList(
```

Comment:
Won't that create some kind of nested orderers?

Reply:
No, it casts the input `list` into a `PackageOrderList`.  Works the same way as doing something like:
``` python
>>> a = [1, 2, 3]
>>> list(list(list(list(a))))
[1, 2, 3]
```